include/dirac_quda.h

```diff
@@ -357,8 +357,7 @@ namespace quda {
      *  @param long_gauge_in Updated long links
      *  @param clover_in Updated clover field
      */
-    virtual void updateFields(cudaGaugeField *gauge_in, cudaGaugeField *fat_gauge_in, cudaGaugeField *long_gauge_in,
-                              cudaCloverField *clover_in)
+    virtual void updateFields(cudaGaugeField *gauge_in, cudaGaugeField *, cudaGaugeField *, cudaCloverField *)
```

Comment:
unnamed function arguments just don't get me excited.

Reply:
Me either.  However, it is the only platform independent way to vanquish unused argument warnings.  Note that once we go C++17, many of the unnamed argument can be vanquished through use of `if constexpr`.